lte/gateway/python/magma/mobilityd/ip_allocator_static.py

```diff
@@ -138,6 +138,7 @@ def _allocate_static_ip(self, sid: str) -> Optional[IPDesc]:
                     ip_addr_info.ip,
                     ip_pool,
                 )
+                return IPDesc()
```

Comment:
The function above will raise an exception so I don't think the new `return` is ever reached.

Reply:
Addressed this.